tests/test_dataset.py

```diff
@@ -84,165 +92,91 @@ def test_dataset(cache, tmpdir, db):
     assert dataset.example == expected_example
 
     # files
-    expected_files = str(len(db.files))
+    expected_files = len(db.files)
     assert dataset.files == expected_files
 
     # formats
-    expected_formats = ', '.join(set([
-        audeer.file_extension(file) for file in db.files
-    ]))
+    expected_formats = sorted(
+        list(
+            set(
+                [
+                    audeer.file_extension(file)
+                    for file in db.files
+                ]
+            )
+        )
+    )
     assert dataset.formats == expected_formats
 
+    # license
+    expected_license = db.license or 'Unknown'
+    assert dataset.license == expected_license
+
     # license link
-    license = db.license or 'Unknown'
     if (
             db.license_url is None
             or len(db.license_url) == 0
     ):
-        expected_license_link = license
+        expected_license_link = None
     else:
-        expected_license_link = f'`{license} <{db.license_url}>`__'
+        expected_license_link = db.license_url
     assert dataset.license_link == expected_license_link
 
-    # name_link
-    expected_name_link = f'`{db.name} <./datasets/{db.name}.html>`__'
-    assert dataset.name_link == expected_name_link
-
     # publication: skipped for now
+
+    # repository
+    expected_repository = pytest.REPOSITORY.name
+    assert dataset.repository == expected_repository
+
     # repository_link : skipped for now
 
     # sampling_rates
-    expected_sampling_rates = ', '.join(set([
-        str(audiofile.sampling_rate(file)) for file in db.files
-    ]))
+    expected_sampling_rates = sorted(
+        list(
+            set(
+                [
+                    audiofile.sampling_rate(file)
+                    for file in db.files
+                ]
+            )
+        )
+    )
     assert dataset.sampling_rates == expected_sampling_rates
 
     # schemes
-    expected_schemes = audbcards.core.dataset.format_schemes(db.schemes)
+    expected_schemes = list(db.schemes)
     assert dataset.schemes == expected_schemes
 
+    # schemes_table
+    expected_schemes_table = [
+        ['ID', 'Dtype', 'Min', 'Labels', 'Mappings'],
+        ['age', 'int', 0, '', ''],
+        ['emotion', 'str', '', 'angry, happy, neutral', ''],
+        ['gender', 'str', '', 'female, male', ''],
+        ['speaker', 'int', '', '0, 1', 'age, gender'],
```

Comment:
It is a little bit unfortunate that we define the expected table here manually as this will of cause break when we change the database used in the test. The problem is that possible code to autogenerate this would also be complicated and not easy to see if it is correct.
Not sure yet, how to best solve this.
For now I would propose to stay with the manual defined expected value.